charts/pvault-server/templates/configmap.yaml

```diff
@@ -66,11 +72,15 @@ data:
     custom_types_enable = {{ .Values.pvault.features.customTypesEnable }}
 
     [kms]
-    uri   = {{ tpl .Values.pvault.kms.uri . | quote }}
-    export_uri  = {{ tpl .Values.pvault.kms.exportUri . | quote }}
     {{- if kindIs "bool" .Values.pvault.kms.allow_local }}
     enable = {{ .Values.pvault.kms.allow_local }}
```

Comment:
`allow_local` null by default, there isn't any other logic checking on the value of dev_mode.

@nirhaas should `kms.enable` be `true` by default and conditionally set to false if `allow_local` is true ?

Reply:
Thanks, fixed.